src/zarr/core/config.py

```diff
@@ -56,6 +59,14 @@ def reset(self) -> None:
         self.clear()
         self.refresh()
 
+    def enable_gpu(self) -> ConfigSet:
+        """
+        Configure Zarr to use GPUs where possible.
+        """
+        return self.set(
```

Comment:
I believe this shows it's covered: https://app.codecov.io/github/zarr-developers/zarr-python/blob/TomAugspurger%2Fzarr-python%3Atom%2Ffix%2Fgpu/src%2Fzarr%2Fcore%2Fconfig.py#L66

Maybe the warning comes from the CPU-only coverage run?